mungegithub/mungers/submit-queue.go

```diff
@@ -381,8 +383,8 @@ func (sq *SubmitQueue) internalInitialize(config *github.Config, features *featu
 			gcs = utils.NewTestUtils("bucket", "logs", overrideUrl)
 		} else {
 			gcs = utils.NewWithPresubmitDetection(
-				utils.KubekinsBucket, utils.LogDir,
-				utils.PullKey, utils.PullLogDir,
+				sq.features.GCSInfo.BucketName, sq.features.GCSInfo.LogDir,
+				sq.features.GCSInfo.PullKey, sq.features.GCSInfo.PullLogDir,
```

Comment:
I don't love the feature interface or how squirrely it is to use features correctly (I wrote it but someone smarter can probably rewrite it better) but I agree it seems like the right thing here...